alicloud/resource_alicloud_security_group.go

```diff
@@ -38,9 +38,16 @@ func resourceAliyunSecurityGroup() *schema.Resource {
 				ForceNew: true,
 			},
 			"inner_access": {
-				Type:     schema.TypeBool,
-				Optional: true,
-				Default:  true,
+				Type:       schema.TypeBool,
+				Optional:   true,
+				Default:    true,
+				Deprecated: "Field 'inner_access' has been deprecated from provider version 1.56.0. Use 'inner_access_policy' replaces it.",
+			},
+			"inner_access_policy": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				Default:      "Accept",
```

Comment:
Please remove the default and set it to computed.
Also, please set a `conflictwith` for `inner_access ` and `inner_access_policy `